test/unit/BondingManager.js

```diff
@@ -1841,6 +1841,20 @@ contract("BondingManager", accounts => {
             assert.equal((await bondingManager.getDelegator(delegator1))[5], currentRound + 1, "should update caller's lastClaimRound to the current round")
         })
 
+        it("fires a ClaimEarnings event", async () => {
```

Comment:
Let's update the event name here to `EarningsClaimed`.

Reply:
fixed in 0ee7707